lametro/signals/handlers.py

```diff
@@ -0,0 +1,12 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+
+from lametro.models import LAMetroPerson, BoardMemberDetails
+
+
+@receiver(post_save, sender=LAMetroPerson)
```

Comment:
Will handle instantiating details for new members moving forward. To backfill, we simply need to call save on all existing board members.